index.html

```diff
@@ -0,0 +1,67 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="johns.css">
+    <link href='https://fonts.googleapis.com/css?family=Poppins' rel='stylesheet'>
+    <title>Document</title>
+</head>
+
+<body>
+    <div class="flexcontainer">
+        <section>
+            <header>
+                <div class="log">
+                    <div class="logo"><img src="images/first/MyLogo.png" alt=""></div>
+                </div>
+                <div class="menu">
+                    <nav>
+                        <ul>
+                            <li class="portfolio item"><a href="#">Portfolio</a></li>
+                            <li class="about item"><a href="#">about </a></li>
+                            <li class="contacts item"><a href="#">contacts</a></li>
+                            <li class="ham"><a href="#"><img src="images/first/main/Union.png" alt=""></a></li>
+
+                        </ul>
+                    </nav>
+                </div>
+            </header>
+            <div class="paragraph">
```

Comment:
Hey @ShahierNashaat  Start the `section` tag after the `header` tag.

Reply:
thanks for the feedback